web-js/4.x/user-add-feature-layer/README.MD

```diff
@@ -0,0 +1,6 @@
+# User Added Feature Layer
+
+This sample shows how you can allow the end user to add their own feature layers to a map. It is just a demo of the input and has no validation.
```

Comment:
What does it mean by the sample has no validation? Could you please add a little more detail about how this sample works? (e.g. that they need to add the URL to the feature service layer id in the text box)